event_sale_extra_info/models/event.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import api, fields, models
+
+
+class EventRegistration(models.Model):
+    _inherit = "event.registration"
+
+    commercial_partner_id = fields.Many2one(
+        "res.partner",
+        "Commercial partner",
+        related="partner_id.commercial_partner_id",
+        readonly=True,
+        store=True,
+        help="Commercial partner related to the chosen partner.",
+    )
+
+    @api.model
+    def _init_commercial_partner_id(self):
```

Comment:
> Why don't do this on a post_init_hook?

What's the benefit?

> Even more, related stored fields are automatically computed at module installation

That's how it should be. In fact I'm doing a stupidity in this method, but I found out that fields were empty after installation without this. I suppose it must be Odoo's bug.